features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql/postgresql.sql

```diff
@@ -2127,7 +2127,7 @@ CREATE TABLE  IF NOT EXISTS AM_KEY_MANAGER (
   DESCRIPTION VARCHAR(256) NULL,
   TYPE VARCHAR(45) NULL,
   CONFIGURATION BYTEA NULL,
-  ENABLED INT NULL,
+  ENABLED BOOLEAN NULL,
```

Comment:
This should be ENABLED BOOLEAN DEFAULT '1'. Please pick appropriate default value 0 or 1.

Reply:
resolved